main

```diff
@@ -90,4 +90,8 @@ module load dust/v0_1
 module load speclite/0.13
 module load QuasarNP/0.1.2
 
+setenv DESI_SPECTRO_CALIB $env(DESI_ROOT)/spectro/desi_spectro_calib/0.2.7
```

Comment:
I should have caught this at PR time, but for the record for the future:  The main environment needs to use desi_spectro_calib/trunk, not a tag, so that it will always have the latest calibrations when processing tonight's data.  I have fixed this directly in main.